client/my-sites/people/team-list/team.jsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' ),
+	debug = require( 'debug' )( 'calypso:my-sites:people:team-list' ),
+	omit = require( 'lodash/omit' );
+
+/**
+ * Internal dependencies
+ */
+var Card = require( 'components/card' ),
+	PeopleListItem = require( 'my-sites/people/people-list-item' ),
+	UsersActions = require( 'lib/users/actions' ),
+	InfiniteList = require( 'components/infinite-list' ),
+	deterministicStringify = require( 'lib/deterministic-stringify' ),
+	NoResults = require( 'my-sites/no-results' ),
+	analytics = require( 'lib/analytics' ),
+	PeopleListSectionHeader = require( 'my-sites/people/people-list-section-header' ),
+	config = require( 'config' );
+
+module.exports = React.createClass( {
+	displayName: 'Team',
+
+	getInitialState: function() {
+		return {
+			bulkEditing: false
+		};
+	},
+
+	isLastPage() {
+		return this.props.totalUsers <= this.props.users.length + this.props.excludedUsers.length;
+	},
+
+	fetchHasCompleted() {
+		return this.props.fetchInitialized && ! this.props.fetchingUsers;
+	},
+
+	render: function() {
+		var key = deterministicStringify( omit( this.props.fetchOptions, [ 'number', 'offset' ] ) ),
+			headerText = this.translate( 'Team', { context: 'A navigation label.' } ),
+			listClass = ( this.state.bulkEditing ) ? 'bulk-editing' : null,
```

Comment:
while this is fine, we could also use `classNames` and make the intention a bit more explicit

```js
const listClass = classNames( {
	'bulk-editing': this.state.bulkEditing,
} );
```